internal/metadataextractor/extractor_test_echoToFile.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
this is not portable if we want to run tests on Windows as well. (I guess the CI will have told you that as well I hope ^^)

Reply:
maybe I could port it to python, but otherwise I knew it wouldn't work on windows ^^'

Reply:
port to go?